content/en/agent/proxy.md

```diff
@@ -412,47 +683,59 @@ If you are using older version of HAProxy, you have to reload or restart HAProxy
 Edit each Agent to point to HAProxy by setting its `dd_url` to the address of HAProxy, for example: `haproxy.example.com`.
 This `dd_url` setting can be found in the `datadog.yaml` file.
 
-`dd_url: http://haproxy.example.com:3834`
+`dd_url: <SCHEME>://haproxy.example.com:3834`
+
+Replace `<SCHEME>` by `https` if you want your data to be encrypted between the Agent and HAProxy, or by `http` otherwise.
```

Comment:
We should add that the scheme needs to match their HAProxy settings

Reply:
I rephrased it here : 41ea6a4696acb87e90d0bef3aa1899e4a3ab4669 WDYT ?